src/test/java/uk/gov/hmcts/reform/civil/service/docmosis/caseprogression/JudgeFinalOrderGeneratorTest.java

```diff
@@ -0,0 +1,114 @@
+import uk.gov.hmcts.reform.civil.documentmanagement.model.CaseDocument;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.PDF;
+import uk.gov.hmcts.reform.civil.enums.caseprogression.FinalOrderSelection;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.caseprogression.FreeFormOrderValues;
+import uk.gov.hmcts.reform.civil.model.common.MappableObject;
+import uk.gov.hmcts.reform.civil.model.docmosis.DocmosisDocument;
+
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDocumentBuilder;
+import uk.gov.hmcts.reform.civil.service.docmosis.DocumentGeneratorService;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType.JUDGE_FINAL_ORDER;
+import static uk.gov.hmcts.reform.civil.service.docmosis.DocmosisTemplates.FREE_FORM_ORDER_PDF;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {
+    JudgeFinalOrderGenerator.class,
+    JacksonAutoConfiguration.class
+})
+public class JudgeFinalOrderGeneratorTest {
+
+    private static final String BEARER_TOKEN = "Bearer Token";
+    private static final String REFERENCE_NUMBER = "000DC001";
+    private static final byte[] bytes = {1, 2, 3, 4, 5, 6};
+    private static final String fileFreeForm = String.format(FREE_FORM_ORDER_PDF.getDocumentTitle(), LocalDate.now());
+    private static final CaseDocument FREE_FROM_ORDER = CaseDocumentBuilder.builder()
+        .documentName(fileFreeForm)
+        .documentType(JUDGE_FINAL_ORDER)
+        .build();
+    @MockBean
+    private UnsecuredDocumentManagementService documentManagementService;
+
+    @MockBean
+    private DocumentGeneratorService documentGeneratorService;
+
+    @Autowired
+    private JudgeFinalOrderGenerator generator;
+
+    @Test
+    void shouldGenerateFreeFormOrder_whenNoneSelected() {
+        when(documentGeneratorService.generateDocmosisDocument(any(MappableObject.class), eq(FREE_FORM_ORDER_PDF)))
+            .thenReturn(new DocmosisDocument(FREE_FORM_ORDER_PDF.getDocumentTitle(), bytes));
+        when(documentManagementService
+                 .uploadDocument(BEARER_TOKEN, new PDF(fileFreeForm, bytes, JUDGE_FINAL_ORDER)))
+            .thenReturn(FREE_FROM_ORDER);
+
+        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
+            .finalOrderSelection(FinalOrderSelection.FREE_FORM_ORDER)
+            .build();
+        CaseDocument caseDocument = generator.generate(caseData, BEARER_TOKEN);
+
+        assertThat(caseDocument).isNotNull();
```

Comment:
could change these to the `assertNotNull()` method like lower down, keep with the junit assertions

Reply:
And done

Also template is 
![image](https://user-images.githubusercontent.com/93932689/228588275-660655b0-6f77-4736-9a15-d6ef5e4ba53f.png)

Reply:
Note I raised some questions over <<claimantReference>> <<claimantReferenceTwo>> and the defendant equivalents, we're unclear on what it should be (believe we aim to clarify on PO review) and also that they are optional fields on create claim journeys, so potentially will never exist to be shown on template